packages/dashboard/src/widgets/StockWidget.tsx

```diff
@@ -21,6 +21,8 @@ export const StockWidget: React.FC = () => {
   const formatNumber = useFormatNumber();
   const { data: expiryData, isLoading: isExpiryLoading } = useStockCounts();
   const { data: itemStatsData, isLoading: isItemStatsLoading } = useItemStats();
+  const [hasExpiryError, setHasExpiryError] = React.useState(false);
```

Comment:
set a local state error and don't show stats if you don't have permission to view them.
Could display text, but I thought in this case, better to just quietly hide the stats.